docker/Dockerfile.deps

```diff
@@ -71,6 +71,14 @@ RUN LIBTIFF_VERSION=4.1.0 && \
     cd && \
     rm -rf /tmp/tiff-${LIBTIFF_VERSION}
 
+RUN OPENJPEG_VERSION=2.3.1 && \
+    cd /tmp && \
+    curl -L https://github.com/uclouvain/openjpeg/archive/v${OPENJPEG_VERSION}.tar.gz | tar -xzf - && \
```

Comment:
Also add it to the `Acknowledgements.txt`.

Reply:
done